### PR TITLE
Added Moralis to Data tools page

### PR DESCRIPTION
**Pull request by filipmartinsson**
Added Moralis as an API provider to the list of data tools in the Polygon documentation

### CODE REVIEW

File: docs/tools/data/moralis.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+
+!!! note "Content disclaimer"
+
+    Please view the third-party content disclaimer [here](https://github.com/0xPolygon/polygon-docs/blob/main/CONTENT_DISCLAIMER.md).
+
+## About Moralis
+**[Moralis](https://moralis.io/?utm_source=polygon-docs&utm_medium=partner-docs)** is a blockchain data platform that provides developers with all the data they need to build better blockchain applications. From NFT data, token data and price data, through to raw blockchain data and RPC nodes, Moralis offers a wide range of products that cover all major crypto and blockchain use cases, and it supports Polygon together with all other major EVM chains.
```
Comment (kmurphypolygon) [inline docs/tools/data/moralis.md:8]:
```suggestion

**[Moralis](https://moralis.io/?utm_source=polygon-docs&utm_medium=partner-docs)** is a blockchain data platform that provides developers with all the data they need to build better blockchain applications. From NFT data, token data and price data, through to raw blockchain data and RPC nodes, Moralis offers a wide range of products that cover all major crypto and blockchain use cases, and it supports Polygon together with all other major EVM chains.
```

File: docs/tools/data/moralis.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+!!! tip "Quickstart"
+
+    Check out our popular **[Youtube channel](https://www.youtube.com/@MoralisWeb3/featured)** for many great tutorials on how to get started.
+
+## Get started
+Get started with Moralis APIs on Polygon by signup up for a [free account](https://admin.moralis.io/register?utm_source=polygon-docs&utm_medium=partner-docs). Then you can check out our [Get Started Guide](https://docs.moralis.io/web3-data-api/evm/get-your-api-key/?utm_source=polygon-docs&utm_medium=partner-docs).
```
Comment (kmurphypolygon) [inline docs/tools/data/moralis.md]:
```suggestion

Get started with Moralis APIs on Polygon by signup up for a [free account](https://admin.moralis.io/register?utm_source=polygon-docs&utm_medium=partner-docs). Then you can check out our [get started guide](https://docs.moralis.io/web3-data-api/evm/get-your-api-key/?utm_source=polygon-docs&utm_medium=partner-docs).
```

File: docs/tools/data/moralis.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+## Moralis APIs
+
+All Moralis APIs have support for Polygon and across all other major EVM blockchains. All endpoints have powerful filtering capabilities. 
+
+!!! info Endpoints
```
Comment (kmurphypolygon) [inline docs/tools/data/moralis.md]:
```suggestion
!!! info "Endpoints"
```

File: docs/tools/data/moralis.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+    Please check out the [<ins>Moralis API Reference</ins>](https://docs.moralis.io/?utm_source=polygon-docs&utm_medium=partner-docs) to see all the available API endpoints.
+
+### Wallet API
+
+With Moralis [<ins>Wallet API</ins>](https://moralis.io/api/wallet/?utm_source=polygon-docs&utm_medium=partner-docs) you can get Wallet balances for tokens, NFTs and native assets, get full wallet history, net worth and a lot more.
```
Comment (kmurphypolygon) [inline docs/tools/data/moralis.md]:
```suggestion
With Moralis [<ins>Wallet API</ins>](https://moralis.io/api/wallet/?utm_source=polygon-docs&utm_medium=partner-docs) you can get wallet balances for tokens, NFTs and native assets, get full wallet history, net worth and a lot more. 
```

File: docs/tools/data/moralis.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+### NFT API
+
+With Moralis [<ins>NFT API</ins>](https://moralis.io/api/nft/?utm_source=polygon-docs&utm_medium=partner-docs) you can get NFT data like collections, owners, prices, images and metadata. 
+
+### Token API
+
+With Moralis [<ins>Token API</ins>](https://moralis.io/api/token/?utm_source=polygon-docs&utm_medium=partner-docs) you can get ERC20 token data like prices, ownership, metadata, transfers, approvals, liquidity, mints and burns. 
+
+### RPC Nodes
```
Comment (kmurphypolygon) [inline docs/tools/data/moralis.md]:
```suggestion
### RPC nodes
```

File: docs/tools/data/moralis.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+Get access to powerful RPC nodes on all major chains with [<ins>Moralis Nodes</ins>](https://moralis.io/nodes/?utm_source=polygon-docs&utm_medium=partner-docs).
```
Comment (kmurphypolygon) [inline docs/tools/data/moralis.md]:
```suggestion
Get access to powerful RPC nodes on all major chains with [<ins>Moralis nodes</ins>](https://moralis.io/nodes/?utm_source=polygon-docs&utm_medium=partner-docs). 
```

**Comment by kmurphypolygon** [comment]
Hi @filipmartinsson thanks so much for your contribution. Please accept my suggested edits.

In order to add this page to the site, you need to create a menu entry in `mkdocs.yaml` at the correct place. 

Once you've done that, and tested it renders correctly, please push the changes and I can merge.

Thank you :)

**Comment by filipmartinsson** [comment]
> Hi @filipmartinsson thanks so much for your contribution. Please accept my suggested edits.
> 
> In order to add this page to the site, you need to create a menu entry in `mkdocs.yaml` at the correct place.
> 
> Once you've done that, and tested it renders correctly, please push the changes and I can merge.
> 
> Thank you :)

Thanks @kmurphypolygon . I've added it to the menu now. Appreciate the help :)

**Review by kmurphypolygon — APPROVED** [review]

**Pull request merged**